test/unit/spec/ForgotPassword_spec.js

```diff
@@ -257,9 +257,11 @@ function (Q, Okta, OktaAuth, Util, AccountRecoveryForm, PrimaryAuthForm, Beacon,
           test.setNextResponse(resChallengeEmail);
           test.form.setUsername('foo');
           test.form.sendEmail();
-          return tick();
+          return tick(test);
```

Comment:
may not be fair to ask but can we ditch `tick`?

Reply:
@haishengwu-okta  I am just adding a check here to verify the UI and not introducing tick here. I think we need to take a SIW wide effort to ditch tick. This test in particular doesnt rely on the tick here.